### PR TITLE
1349: support heic images

### LINKED ISSUE #1349: HEIF support so iPhone/iPad owners can naively upload to Apostrophe
A heads up: iphones now snap by default in HEIF format. People uploading naively from iphones/ipads to Apostrophe will have trouble as a result until we figure out how to handle HEIF in uploadfs. Since most browsers are going to go “whuh?” I would recommend converting it. One wonders if there is even an open source toolchain with a suitable license for converting it.

Since we don't have a mobile UI yet this is mostly an issue after they sync that photo to some other device for now, or on sites that have frontend code that accepts submissions and utilizes apostrophe-attachments (which is pretty common).

### PR DESCRIPTION
**Pull request by glmaljkovich**
This adds a [middleware](https://github.com/boutell/heic-to-jpeg-middleware) to handle heic/heif image uploads coming from recent iPhones.

closes #1349

### CODE REVIEW

**Comment by glmaljkovich** [comment]
[heic-to-jpeg-middleware](https://www.npmjs.com/package/heic-to-jpeg-middleware) npm package version needs to be bumped to `2.0.0` before we can test this.

**Comment by abea** [comment]
I think @boutell will give this a review, but we will ask that you update the changelog, in case you want to get that over with. Thanks for the contribution!

File: lib/modules/apostrophe-attachments/lib/routes.js [hunk 1]
```diff
@@ -2,26 +2,30 @@ var _ = require('@sailshq/lodash');
...
-    return self.accept(req, file, function(err, file) {
-      if (err) {
-        return next(err);
-      }
-      if (req.query.html) {
-        res.setHeader('Content-Type', 'text/html');
+  self.apiRoute('post', 'upload',
+    self.middleware.canUpload,
+    self.apos.middleware.files,
+    self.apos.middleware.heicImages,
```
Comment (glmaljkovich) [inline lib/modules/apostrophe-attachments/lib/routes.js:8]: This is the only change on this file, the rest is linting

**Comment by boutell** [comment]
This needs an actual test with an heic file

**Comment by boutell** [comment]
I also need to publish the change on the actual middleware, which is a boutell repo thing

**Comment by boutell** [comment]
The middleware and a corresponding apostrophe update will be published next Wednesday. Thanks for your contributions!

**Comment by glmaljkovich** [comment]
You're welcome!

**Pull request merged**